README.md

```diff
@@ -60,29 +60,36 @@ bedrock.events.on('bedrock-mongodb.ready', function(callback) {
 bedrock.start();
 ```
 
-Below is an example demonstrating the use of a distributed ID generator.
+## Configuration
 
-```js
-var database = require('bedrock-mongodb');
+For documentation on database configuration, see [config.js](./lib/config.js).
 
-database.getDistributedIdGenerator('mynamespace', function(err, idGenerator) {
-  if(err) {
-    console.error('Error', err);
-    return;
-  }
-  idGenerator.generateId(function(err, id) {
-    if(err) {
-      console.error('Error', err);
-      return;
-    }
-    console.log('ID generated', identifier);
-  });
-});
-```
+### Connecting and Authenticating
+MongoDB's documentation offers tons of great examples on how to authenticate
+using a myriad number of connection strings.
 
-## Configuration
+[Mongo Node 3.5 Driver connect docs](http://mongodb.github.io/node-mongodb-native/3.5/tutorials/connect/)
 
-For documentation on database configuration, see [config.js](./lib/config.js).
+[Mongo Node 3.5 Driver atlas docs](https://docs.mongodb.com/drivers/node#connect-to-mongodb-atlas)
+
+You can also connect to access-enabled mongo servers using some small changes to the
+`config.mongodb.connectOptions`:
+```js
+const {config} = require('bedrock');
+config.mongodb.username = 'me';
+config.mongodb.password = 'password';
+const {connectOptions} = config.mongodb;
+// this optional and only required if connecting to a replicaSet
+connectOptions.replicaSet = 'my_provider_replica_set';
+// optional, but required in production by many providers
+connectOptions.ssl = true;
+// optional but required is your provider requires tls 
+connectOptions.tls = true;
+// the `authSource` option replaces the older `authDB` option
+connectOptions.authSource: 'my_provider_auth_db'; 
+// an authSource should be specified else it will be the `mongodb.name`
+connectOptions.authSource = 'admin';
```

Comment:
```suggestion
// the `authSource` option replaces the older `authDB` option
// it should be specified or else it will be the `mongodb.name`
connectOptions.authSource = 'my_provider_auth_db'; 
connectOptions.authSource = 'admin';
```